evap/evaluation/models.py

```diff
@@ -791,7 +791,7 @@ def unpublish(self):
 
     @classmethod
     def state_to_str(cls, state):
-        return cls.STATE_STR_CONVERSION[state]
```

Comment:
Instead of translating it here, please add the underscore call to all strings above in `STATE_STR_CONVERSION`. Then "editor_approved" and "in_evaluation" will be added to the translated strings.

Reply:
Now included with 545a75e :+1: